priv/emqx.schema

```diff
@@ -385,8 +385,8 @@ end}.
 
 %% RPC SSL server port.
 {mapping, "rpc.enable_ssl", "gen_rpc.ssl_server_port", [
-  {default, 5369},
-  {datatype, integer}
+  {default, "5369"},
```

Comment:
Should be false by default?

Reply:
That's maybe a better default.

I didn't change that to make the change less "invasive".  I wonder how the tests are passing in CI, I don't recall seeing them create certificates :thinking:

Reply:
It is wrong that the same port is used for ssl and tcp.